test/index.html

```diff
@@ -15,12 +15,12 @@
 <body>
   <script>
     WCT.loadSuites([
-      'basic.html',
       'basic.html?dom=shadow',
       // To enable `useNativeCSSProperties`, the native Shadow DOM and lazy
       // registration flags must also be enabled.
       // https://github.com/Polymer/polymer/blob/ff6e884ef4f309d41491333860a8bc9c2f178696/src/lib/settings.html#L55
-      'basic.html?dom=shadow&lazyRegister=true&useNativeCSSProperties=true'
+      'basic.html?dom=shadow&lazyRegister=true&useNativeCSSProperties=true',
```

Comment:
Is there any particular reason as to why we are running tests with these extra flags?